libraries/chain/market_evaluator.cpp

```diff
@@ -258,17 +267,39 @@ void_result call_order_update_evaluator::do_apply(const call_order_update_operat
       else
       {
          const auto call_obj  = d.find(call_order_id);
+         // we know no black swan event has occurred
          FC_ASSERT( call_obj, "no margin call was executed and yet the call object was deleted" );
-         //edump( (~call_obj->call_price) ("<")( _bitasset_data->current_feed.settlement_price) );
-         // We didn't fill any call orders.  This may be because we
-         // aren't in margin call territory, or it may be because there
-         // were no matching orders.  In the latter case, we throw.
-         GRAPHENE_ASSERT(
-            ~call_obj->call_price < _bitasset_data->current_feed.settlement_price,
-            call_order_update_unfilled_margin_call,
-            "Updating call order would trigger a margin call that cannot be fully filled",
-            ("a", ~call_obj->call_price )("b", _bitasset_data->current_feed.settlement_price)
-            );
+         if( d.head_block_time() <= HARDFORK_CORE_583_TIME )
+         {
+            //edump( (~call_obj->call_price) ("<")( _bitasset_data->current_feed.settlement_price) );
```

Comment:
Please remove this commented out edump() and add a TODO to remove the check after HF has passed.